content/01.synopsis.md

```diff
@@ -1,10 +1,10 @@
 ## Abstract {.page_break_before}
 
-The advent of high-throughput profiling methods such as genomics and other technologies has accelerated basic research and made deep molecular characterization of patient samples routine. 
-These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
+The advent of high-throughput profiling methods (such as genomics or imaging) has accelerated basic research and made deep molecular characterization of patient samples routine. 
+They provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
```

Comment:
This has swung too far in the other direction as to be somewhat unclear, in my opinion.
```suggestion
These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
```